lib/rules/no-invalid-link-text.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+const DISALLOWED_LINK_TEXT = [
+  // WCAG F84-Provided Examples
+  'click here',
+  'more info',
+  'read more',
+  'more',
+];
+module.exports = class NoInvalidLinkText extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (node.tag !== 'a') {
+          return;
+        }
+        if (AstNodeInfo.isLinkElement(node)) {
+          // Extract the text content(s) from the TextNode child(ren)
+          const nodeChildren = AstNodeInfo.childrenFor(node);
+          const textChildren = nodeChildren.filter(child => AstNodeInfo.isTextNode(child));
+          const linkTexts = textChildren.map(linkText => linkText['chars'].toLowerCase().trim());
```

Comment:
If `textChildren.length !== nodeChildren.length` we have to assume that the other dynamic bits have added the additional descriptive content.

Can you add additional "good" tests for:

```hbs
<a href="/asdf">{{someGreatDescription}} click here</a>
```